lib/contestant.rb

```diff
@@ -0,0 +1,28 @@
+class Contestant
+  attr_reader :age, :state_of_residence, :spending_money, :game_interests
+
+  def initialize(contestants)
```

Comment:
This is a poor naming choice. This should be called `contestant_info` or `contestant_attributes`